### PR TITLE
Use theme assets

### PR DESCRIPTION
**Pull request by lennym**
Instead of laoding hof-frontend-toolkit in as a separate dependency

### CODE REVIEW

**Comment by JoeChapman** [comment]
moar rebase

File: assets/scss/app.scss [hunk 1]
```diff
@@ -1,8 +1,7 @@
 $path: "../img/";
 
-@import "hof-frontend-toolkit";
+@import "hof-theme-govuk";
 
-@import "./alert.scss";
 @import "./icons.scss";
 @import "./validation.scss";
 @import "./print.scss";
```
Comment (JoeChapman) [inline assets/scss/app.scss]: Are these stylesheets still required, are they not common?

Comment (lennym) [inline assets/scss/app.scss]: They might be, I don't actually know. 

I just know that this particular PR doesn't make them any more common than they previously were.

`alerts.scss` was definitely rolled into `hof-frontend-toolkit` a few weeks ago (by me) so I removed that one, but I don't know the status of the others. I'll take a look.

**Comment by lennym** [comment]
Re-re-based.

File: assets/scss/app.scss (hunk 1 above)
Comment (lennym) [inline assets/scss/app.scss]: I had a look and there's no duplication of code between here and `hof-frontend-toolkit`, so yes, these stylesheets are still required.

**Pull request merged**